ColorMap.cpp

```diff
@@ -0,0 +1,29 @@
+#include "ColorMap.h"
+#include <sstream>
+#include <utility>
+
+ColorMap::ColorMap() {
+    majorColors = { "White", "Red", "Black", "Yellow", "Violet" };
+    minorColors = { "Blue", "Orange", "Green", "Brown", "Slate" };
+}
+
+std::vector<std::pair<std::string, std::string>> ColorMap::getColorPairs() const {

```

Comment:
Nice function to split the sequencing of color pairs from the actual printing :)